frontend-react/src/pages/submissions/SubmissionsTable.tsx

```diff
@@ -49,6 +110,39 @@ function SubmissionsTable() {
                         })}
                     </tbody>
                 </table>
+                {submissions.length === 0 && (
+                    <p>There were no results found.</p>
```

Comment:
Optional: I prefer "No more results found" but we can always have Drew tweak the wording...

Reply:
the reason why i chose this wording is because you can come to the page with no results.
i guess we can change the wording based on if there is a `cursor` or not...
what do you think? :smile:

Reply:
Heh. That might be a nice detail.